src/Newbe.BookmarkManager/Components/ControlPanel.razor

```diff
@@ -221,4 +226,35 @@
         await LoadUserOptions();
     }
 
+    private Task OnChangePinyinFeatureAccessToken(string accessToken)
+    {
+        try
+		{
+			_modal.PinyinFeature.Expire = WebHelper.GetJwtExp(accessToken);
+		}
+		catch (Services.Common.AccessTokenInvalidException e)
+		{
+			_modal.PinyinFeature.Expire = null;
+			_modal.PinyinFeature.AccessToken = string.Empty;
```

Comment:
If this event is fired, AccessToken would change automatically, am I right?
So, I think it is ok to remove this line.